README.rst

```diff
@@ -32,20 +79,24 @@ When you're going to release a new version of the stack, you should:
 
   Example::
 
-    git tag -f 1.1
-    git tag 1.1-20160731
+    git tag -f 1.1-py3
+    git tag 1.1-py3-20160804
+
+  Tag ``1.1-py3-latest`` will be built automatically from ``branch-1.1-py3``, no need to define a tag manually.
 
 3. Push the changes and the tags to the repo::
 
-    git push -f origin master <tag1> <tag2>
+    git push -f origin master <tag1> <tag2> <tag3>
```

Comment:
`git push -f origin <branch> <tag1> <tag2> <tag3>`

I'm not sure what kind of development will go in `master` - it seems to me that we will do development in parallel in 3 major branches.